events/class_tests/create_meeting_form.py

```diff
@@ -0,0 +1,181 @@
+import pytest
+from users.models import User
+from events.models import Event, EventParticipant, OptionalMeetingDates
+from events.views import CreateMeetingView
+from events.tests import valid_event_data, EVENT_DATE_TIME_START, EVENT_DATE_TIME_END  # noqa: F401
+from events.forms import EventCreationForm
+from datetime import datetime
+from django.utils import timezone
+from pytest_django.asserts import assertTemplateUsed
+
+OPTIONAL_MEETING_DATE_START = datetime(2023, 3, 24, 12, 12, 12, 0, tzinfo=timezone.utc)
+OPTIONAL_MEETING_DATE_END = datetime(2023, 3, 24, 14, 12, 12, 0, tzinfo=timezone.utc)
+OPTIONAL_MEETING_DATE_START_IN_THE_PAST = "2020-11-11 10:10"
+OPTIONAL_MEETING_DATE_END_IN_THE_PAST = "2020-11-11 11:40"
+
+MEETING_CREATION_URL = '/event/meeting/'
+MEETING_CREATION_HTML_PATH = "meetings/create_meeting.html"
+
+
+@pytest.fixture
+def valid_optional_meeting_data():
+    return {
+        'optional_meetings-TOTAL_FORMS': '1',
+        'optional_meetings-INITIAL_FORMS': '0',
+        'optional_meetings-0-date_time_start': OPTIONAL_MEETING_DATE_START,
+        'optional_meetings-0-date_time_end': OPTIONAL_MEETING_DATE_END,
+    }
+
+
+@pytest.fixture
+def valid_participant_data():
+    return {
+        'participants-TOTAL_FORMS': '1',
+        'participants-INITIAL_FORMS': '0',
+        'participants-0-participant_email': 'testUser2@mta.ac.il',
```

Comment:
You can write the email- 'testUser2@mta.ac.il' as constant and use it in lines 35, 66 and 168